cypher-editor-support/src/completion/rules/ruleParameter.js

```diff
@@ -0,0 +1,11 @@
+import * as CompletionTypes from '../CompletionTypes';
+
+export default (element) => {
+  const text = element.getText();
+  if (text === '$') {
```

Comment:
I can't come up with an example when it breaks.

But it feels that checking for a '$' sign is not enough. We should be more specific in this rule, that we are working with parameter.

More over, we already have rule that covers parameters.
Adding dollar sign to schema might solve this issue and this particluer rule will not be needed at all.